cycledash/auth.py

```diff
@@ -65,6 +65,24 @@ def is_anonymous(self):
     def get_id(self):
         return unicode(self['id'])
 
+class AnonymousUser(dict):
```

Comment:
You could inherit from User and from get_id (which, IIRC, should return unicode). If not that, make sure get_id returns unicode if it should (cf. above User).